### PR TITLE
Create richardson.md

### PR DESCRIPTION
**Pull request by LynnRichardson**
my file

### CODE REVIEW

File: richardson.md [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a test
```
Comment (ShawnShealy) [inline richardson.md:1]: Looks great to me!  :+1:

Comment (SangySantharam) [inline richardson.md:1]: @LynnRichardson testing line level comments

Comment (githubteacher) [inline richardson.md:1]: @LynnRichardson What do you want to test? :trollface:

Comment (jdunnett) [inline richardson.md:1]: @LynnRichardson testing

Comment (hs186016) [inline richardson.md:1]: @LynnRichardson Good work!!

**Comment by bhughes2** [comment]
testing

File: richardson.md (hunk 1 above)
Comment (lakshgupta) [inline richardson.md:1]: :+1:

Comment (sweetfisheric) [inline richardson.md:1]: :trollface:

**Comment by Smarzulla** [comment]
Can I have your autograph?

File: richardson.md (hunk 1 above)
Comment (ringhilterra) [inline richardson.md:1]: :tada:

Comment (ItaipuTechnology) [inline richardson.md:1]: Teacher's Pet!!! Good Job!

**Comment by ap120266** [comment]
@LynnRichardson  sweet.

**Comment by LynnRichardson** [comment]
test complete comment

**Pull request merged**